packages/GenderSelection/src/GenderSelection.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import { Story, Meta } from '@storybook/react/types-6-0';
+
+import { GenderSelection } from './index';
+import { GenderProps,SelectOptions } from './GenderSelection';
+export default {
+    title: 'Components/GenderSelection',
+    component: GenderSelection,
+} as Meta;
+
+const optionArr = [
+    {
+        label: 'Male',
+        value: 'male'
+    }, {
+        label: 'Female',
+        value: 'female'
+    }, {
+        label: '+ Custom',
+        value: 'custom'
+    }
+]
+
+export const GenderSelectionExample = () => {
+    const [selectedValue, setSelectedValue] = useState('');
+    const [customGender, setCustomGender] = useState('');
+    const [options, setOptions] = useState<SelectOptions>(optionArr)
+    return <GenderSelection
+        options={options}
+        value={selectedValue}
+        placeholder={'Select Gender'}
+        onChange={(e: any) => setSelectedValue(e.target.value)}
+        customGenderOnChange={(e: any) => setCustomGender(e.target.value)}
+        customGenderValue={customGender}
+        onEnterKeyPressed={() => {
+            console.log('save in DB')
```

Comment:
Did you intend to leave this console log?

Reply:
Yes we did, these are just storybook examples that to help let a dev know this is when you should save in a DB

Reply:
okay!